javascript/random-color.js

```diff
@@ -0,0 +1,28 @@
+const darkColorsArr = [
```

Comment:
Change this to use the self invoking function approach I showed you last week and shared on codepen